tests/test_unasync.py

```diff
@@ -47,6 +47,8 @@ def test_unasync(tmpdir, source_file):
 
 def test_build_py(tmpdir):
 
+    tmpdir = str(tmpdir)
```

Comment:
Instead of converting it to a string, you can replace `os.path.join(tmpdir, "example_pkg")` with `tmpdir / "example_pkg". tmpdir is a pathlib2 object, which is a PyPI version of https://docs.python.org/3/library/pathlib.html that works on Python 2 too.